src/machine-learning/machine-learning.ts

```diff
@@ -288,22 +304,26 @@ export interface TFLiteModel {
 }
 
 
-/**
- * A Firebase ML Model input object
- */
-export class ModelOptions {
-  public displayName?: string;
-  public tags?: string[];
 
-  public tfliteModel?: { gcsTfliteUri: string; };
-}
 
+function getMaskFromOptions(options: ModelOptions): string[] {
```

Comment:
Can we use the existing `generateUpdateMask()` function in `utils/index.ts`?

Reply:
No, utils.generateUpdateMask gives you camelCase, we need snake_case. The mask is required to matched the proto field name and protos are in snake_case. We could change the ModelOptions definition to be in snake_case, but I think that would not be a good experience for Node.js developers who are used to camelCase field names.